src/_data/whats-new.yml

```diff
@@ -21,9 +27,69 @@ entries:
   type: Major update
   date: December 7, 2020
   link: https://github.com/magento/merchdocs/pull/914
+- description: Added the ability to [monitor and manage](https://docs.magento.com/user-guide/system/catalog-sync.html)
```

Comment:
Looks like a duplicate

Reply:
Thanks for catching -- fixed. The tool did not like that we put those P-rex items in ahead of time. Thought I caught all of those.